modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-cloud/clientConfiguration.mustache

```diff
@@ -31,7 +31,7 @@ public class ClientConfiguration {
   private String {{{name}}}Password;
 
   @Bean
-  @ConditionalOnProperty(name = "{{{title}}}.security.{{{name}}}.username")
+  @ConditionalOnProperty(name = "{{#lowercase}}{{{title}}}{{/lowercase}}.security.{{{name}}}.username")
```

Comment:
shouldn't the lambda be invoked with `{{#lambda.lowercase}}`, as described in https://github.com/OpenAPITools/openapi-generator/blob/master/docs/templating.md#mustache-lambdas ?